lib/config_helper.rb

```diff
@@ -8,7 +8,7 @@ def setup_action_mailer(config)
     if FeatureFlipper.send_email?
       config.action_mailer.delivery_method = :govdelivery_tms
       config.action_mailer.govdelivery_tms_settings = {
-        auth_token: ENV['GOVDELIVERY_TOKEN'],
+        auth_token: ENV['GOV_DELIVERY_TOKEN'],
         api_root: "https://#{FeatureFlipper.staging_email? ? 'stage-' : ''}tms.govdelivery.com"
```

Comment:
We've got the `GOV_DELIVERY_SERVER` in place with https://github.com/department-of-veterans-affairs/devops/pull/907 but I don't know if it makes sense to update in this PR.